update_keb_installation.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2021 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Update Kebechet installation details in DB during webhook workflows."""
+
+import os
+from tempfile import TemporaryDirectory
+from contextlib import contextmanager
+
+
+import yaml
+import json
+import requests
+from github import Github
+
+from thoth.storages import GraphDatabase
+from thamos.cli import _load_files
```

Comment:
It would be good to move this to the lib part of thamos so we know it is used elsewhere (without underscore in the name as it will not be public anymore).